djangocms_pageadmin/admin.py

```diff
@@ -299,7 +306,7 @@ def duplicate_view(self, request, object_id):
             page_content=obj,
             initial={
                 "site": obj.page.node.site,
-                "slug": obj.page.get_slug(obj.language),
+                "slug": self._get_slug(obj),
```

Comment:
What is the use case and or requirement for changing the slug? I can see why it would be useful but there now seems to be code that doesn't yet do anything?

Reply:
When updating the timestamp in the plugins it would make sense update the slug with the same timestamp to keep consistency. Hence this change.

Reply:
Ie the idea is that it can provide an interface for when a developer inherits from this pageadmin.